qualtran/bloqs/multi_control_multi_target_pauli.py

```diff
@@ -137,6 +140,160 @@ def _has_unitary_(self) -> bool:
         return True
 
 
+@frozen
+class MultiControlX(Bloq):
+    r"""Implements multi-control, single-target X gate as a bloq using $n-1$ clean ancillas.
+
+    References:
+        [Constructing Large Controlled Nots]
+        (https://algassert.com/circuits/2015/06/05/Constructing-Large-Controlled-Nots.html)
+    """
+
+    cvs: Tuple[int, ...] = field(converter=lambda v: (v,) if isinstance(v, int) else tuple(v))
+
+    @cached_property
+    def signature(self) -> 'Signature':
+        if len(self.cvs) > 0:
```

Comment:
just philosophically, I think we should mandate that `cvs` must be greater than 0